tiltak/src/main/kotlin/no/nav/amt/tiltak/tiltak/services/TiltakServiceImpl.kt

```diff
@@ -1,17 +1,25 @@
 package no.nav.amt.tiltak.tiltak.services
 
+import com.github.benmanes.caffeine.cache.Caffeine
 import no.nav.amt.tiltak.core.domain.tiltak.Tiltak
 import no.nav.amt.tiltak.core.port.TiltakService
+import no.nav.amt.tiltak.tiltak.dbo.TiltakDbo
 import no.nav.amt.tiltak.tiltak.repositories.TiltakRepository
 import no.nav.amt.tiltak.tiltak.utils.UpdateStatus
 import org.springframework.stereotype.Service
 import java.util.*
+import java.util.concurrent.TimeUnit
 
 @Service
 open class TiltakServiceImpl(
 	private val tiltakRepository: TiltakRepository
 ) : TiltakService {
 
+	private val tiltakCache = Caffeine.newBuilder()
+		.expireAfterWrite(12, TimeUnit.HOURS)
```

Comment:
Betyr ikke dette at endringer potensielt ikke kommer fram før etter 12 timer, hvis tiltaket allerede ligger i cache?

Reply:
Jo, men siden tiltak omtrent aldri endrer seg så er det ikke et problem

Reply:
Kunne i teorien brukt 1 time også siden et også holder for caching

Reply:
Vi trenger caching ja, det gjøres MANGE requests for å hente tiltak fra databasen

Reply:
ok, det er forsåvidt en funksjonell sak da som vi kanskje burde si fra om. Det kan bli vanskelig å teste hvis man skal vendte 12 timer før man ser endringen, så kan hende det er lurt å sette den lavt til å begynne med